cmd/cosign/cli/sign.go

```diff
@@ -100,6 +100,10 @@ EXAMPLES
 				if !oneOf(*kmsVal, *key) {
 					return &KeyParseError{}
 				}
+			} else {
```

Comment:
Ah, I think the issue is that the test was covering the duplicated check below, not up here.